api-lib/HttpError.ts

```diff
@@ -97,7 +99,7 @@ export function errorResponseWithStatusCode(
         scope.addBreadcrumb({
           category: 'query-path',
           message: error.details.path,
-          level: Sentry.Severity.Debug,
+          level: SEVERITY_DEBUG,
```

Comment:
Try this instead, so we can avoid the unneeded typecast on line 16:
```suggestion
          level: SeverityLevels.Debug,
```